src/integrationTest/java/com/mongodb/hibernate/id/MongoIdFieldNameIntegrationTests.java

```diff
@@ -94,6 +99,13 @@ void testEntityWithIdColumnAnnotationWithNameElementIdentical(SessionFactoryScop
             movie.id = 1;
             session.persist(movie);
         });
+        assertCollectionContainsOnly(BsonDocument.parse("{_id: 1}"));
+    }
+
+    private void assertCollectionContainsOnly(BsonDocument expectedDoc) {
```

Comment:
This looks good, but I imagine the same exact thing will be needed in many integration tests.  Is there a common place we can move this (and other future similar methods) that makes sense?

Reply:
We are working on a swiss-knife JUnit extension to minimize such boilerplate to minimal (https://github.com/mongodb/mongo-hibernate/pull/62). When it is merged, we simply add the following one-liner to inject `MongoCollection`, so we can partly alleviate the issue.

Other remaining duplicated logic is simply grabbing documents from a collection and it seems trivial, but definitively we could eliminate that as well by introducing some customized asserter somewhere.